src/App.js

```diff
@@ -5,9 +5,7 @@ import ThingsToCarry from "./components/ThingsToCarry";
 import SearchBar from "./components/SearchBar/SearchBar";
 import { geolocation } from "./api/geolocation";
 import logo from "./mlh-prep.png";
-import { MapContainer, TileLayer, Marker, Popup, useMapEvents } from "react-leaflet";
```

Comment:
Why have these components been removed? These have already been merged into the project and need to be put back.